backend/apid/graphql/schema/entity.graphql

```diff
@@ -35,6 +35,12 @@ type Entity implements Node {
   "All events associated with the entity."
   events(orderBy: EventsListOrder = SEVERITY): [Event!]!
 
+  "isSilenced return true if the entity has any silences associated with it."
+  isSilenced: Boolean!
+
+  "All silences associated with the entity."
+  silencedBy: [Silenced!]!
```

Comment:
I'm having a hard time wrapping my head around the naming. I'm not sure if this is something you've come up with or it's established, so forgive any ignorance.

`silencedBy` makes it seem like it's an array of sources that have silenced this entity. Is that the case? Or are we listing the silenced events that belong to this entity?

Reply:
> I'm having a hard time wrapping my head around the naming. I'm not sure if this is something you've come up with or it's established, so forgive any ignorance.

This is something I've come up with.

> `silencedBy` makes it seem like it's an array of sources that have silenced this entity. Is that the case? Or are we listing the silenced events that belong to this entity?

The former is the intent, however, I can see your point about it being unclear. Another possibility that the field's name might suggest is that it may be a list of user's that silenced the entity.

I think I may switch it to `silences` to match the other relationships and expand the field's description for clarity.

Reply:
If it's the former then I think it's clear enough, unless you really want to switch it. I just don't have the same context around the relationships and my monkey brain was just wanting to be clear.